src/1_utils.js

```diff
@@ -1206,14 +1206,27 @@ utils.getCurrentUrl = function() {
 	return utils.isIframeAndFromSameOrigin() ? window.top.location.href : window.location.href;
 };
 
+utils.convertValueToString = function(value) {
+	if (utils.validateParameterType(value, 'object')) {
+		return safejson.stringify(value);
+	}
+	if (utils.validateParameterType(value, 'array')) {
+		return safejson.stringify(value);
+	}
```

Comment:
It's an extension of the early return pattern. I like it because it reduces mental juggling when reading and debugging code. Let's say I get to this block:
```
        if (utils.validateParameterType(value, 'object')) {
		return safejson.stringify(value);
	}
```
and I have an example in mind that is an object type value, then this is all the code I have to read to continue following the code path. Similarly, when debugging, I can put a breakpoint in the return line here and know that we are looking at an object. 

I'd say this is a personal preference, and there's no right or wrong here. I think I'd lean more towards combining the two conditionals if I did the following, which is easy enough to read:
```
const isObject = utils.validateParameterType(value, 'object');
const isArray = utils.validateParameterType(value, 'array');
if (isObject || isArray) {
  return safejson.stringify(value);
}
```

A similar situation where I have stronger opinions about though (because it's AND) is the following:

I dislike:
```
if (!A && B) {
  return 'first';
}
return 'second'
```

I prefer:
```
if (A) {
  return 'second';
}
if (!B) {
  return 'second';
}
return 'first';
```